lib/email/index.js

```diff
@@ -37,3 +37,19 @@ export const sendResetPassworEmail = (to, token) => {
 
   return sendEmail(config.emails.from, to, subject, textMessage, html)
 }
+
+// send an email after user send request that he want to join to the group.
+export function sendRequestToJoinToGroupEmail(to, type, group) {
+  const linkToGroup = `${config.url}/group/${type}/${group.slug}`
+
+  const textMessage = `Some user want to be a member of ${group.title} group. You can follow the link to check the group members`
+  const subject = `Some user want to be a member of ${group.title} group`
```

Comment:
A user wants to join ${group.title}. Go to the <a href="link" title="user's administration page">user's administration page</a> to approve or deny the membership.

Subject: A user wants to join ${group.title}